lib/similarity_utils.py

```diff
@@ -289,3 +291,19 @@ def find_actor_two_datasets(data1: pd.DataFrame, data2: pd.DataFrame, actor_id:
     else:
         print("W żadnym z tych dwóch zbiorów nie ma wybranego aktora")
         return None
+
+
+def get_ranking_for_actor_data_row(ranking_data_source: pd.DataFrame, actor_data_row: pd.DataFrame, ranking_length: int = 5, reduced_dataset: bool = False) -> List[str]:
+    """metoda dla ramki danych, wiersza wybranego aktora oraz (opcjonalnie) długości rankingu, zwraca listę id aktorów najbardziej
+    podobnych do wybranego aktora"""
+    actor_pandas_row = prepare_pandas_row(actor_data_row)
+    ids, similarities = similarity_one_vs_all(ranking_data_source, actor_pandas_row, reduced_dataset)
+    return select_top_similiar(ids, similarities, ranking_length)[0]
+
+
+def get_ranking_for_actor_data_row_new(ranking_data_source: pd.DataFrame, actor_data_row: pd.DataFrame, ranking_length: int = 5, reduced_dataset: bool = False) -> List[str]:
+    """metoda dla ramki danych, wiersza wybranego aktora oraz (opcjonalnie) długości rankingu, zwraca listę id aktorów najbardziej
+    podobnych do wybranego aktora"""
+    actor_pandas_row = prepare_pandas_row(actor_data_row)
+    ids, similarities = similarity_one_vs_all_new(ranking_data_source, actor_pandas_row, reduced_dataset)
+    return select_top_similiar(ids, similarities, ranking_length)[0]
```

Comment:
zmień nazwy zmiennych `actor_data_row` i `actor_pandas_row`. powinny one być na odwrót

Reply:
Jeżeli dobrze zrozumiałem, to też zrobione